examples/ets_example.ipynb

```diff
@@ -494,9 +494,9 @@
  ],
```

Comment:
if this file is not essential for this PR, probably we can check it out? it seems that it has a conflict with current dev-ktrx.